### DISCUSSION TITLE
Will ${{ placeholders }} work from other Strings?

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#25326] [Actions]
<p>I want to use the <a href="https://github.com/marketplace/actions/label-commenter" rel="nofollow noopener">label-commenter</a> GitHub Action to automatically post canned responses depending on the label provided.</p>
<p>My question now is, if I would f.e. add <code>${{ github.event.sender.login }}</code> to the response String - which corresponds to who created the issue - would GitHub still parse this String or treat it as a literal one?</p>
<p>My guess would be no and that the placeholder would be displayed as is, but I better want to ask here first.</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/Andre601">@Andre601</a>,</p>
<p>I noticed that you also have opened an issue ticket (<a href="https://github.com/peaceiris/actions-label-commenter/issues/152" rel="nofollow noopener">peaceiris/actions-label-commenter#152</a>) , and this question seems has been solved by using the <strong>v1.4.0-0</strong> of the action.</p>
<pre><code class="lang-yaml">- name: Label Commenter
  uses: peaceiris/actions-label-commenter@v1.4.0-0
  with:
    github_token: ${{ secrets.GITHUB_TOKEN }}
</code></pre>
<p>I also tested this version, and it can work fine.</p>
